Answers.md

```diff
@@ -1 +1,23 @@
 
+1. Describe the biggest difference between `.forEach` & `.map`.
+.forEach doesn't return anything. It just calls a callback function for each item in an array. .map returns a new array.
+
+2. What is the difference between a function and a method?
+A method needs an object whereas a function does not.
+
+3. What is closure?
+functionA(){
+    variable1
+    functionB(){
+        variable1
+        variable2
+    }
+    functionC(){
+        variable1
+        variable3
+    }
+}
+4. Describe the four rules of the 'this' keyword.
+
```

Comment:
THIS is actually a complicated YET super important concept, I would highly recommend reviewing it for your own knowledge.